packages/cli/src/bin/cli.js

```diff
@@ -14,7 +15,22 @@ if (os.platform() === "win32") {
 // TODO check current vs required node version (package.json engines)
 // const requiredNodeVersion = require("../package.json").engines.node;
 
-import { Command } from "commander";
+// ask the user for permission to send anonymous telemetry
+if (process.env.FLOWERSHOW !== "dev") {
+  const { allow } = await inquirer.prompt([
```

Comment:
Does this prompt every time? or just the first time.

IMO i think for now we could just default this and not prompt. We aren't collecting *any* personal info etc so i don't think there is an issue collecting info and i don't think we need to warn. So in fact i suggest removing for now.